version1/02 - Object types.md

```diff
@@ -566,27 +566,29 @@ The following notation is used in the tables below:
 | M707      | number    | filstoerrelse          |    x     |      x      |               |  |  sm  |  x   |   x   |  x   |      |        |                                                                    |
 | M218      | string    | referanseDokumentfil   |    x     |             |               |  |  x   |  x   |   x   |  x   |      |        | <p>&bull; unique identifier of a document stored in the system</p> |
 |           |           |                        |          |             |               |  |      |      |       |      |      |        |                                                                    |
-|           | ref       | refDokument            |    x     |             |               |  |      |  x   |   x   |      |  x   |   x    | &nbsp;                                                             |
+|           | ref       | refDokument            |    x     |             |               |  |      |  x   |   x   |      |  x   |   x    |                                                                    |
+|           | ref       | refEksternId           |    x     |             |               |  |      |      |   x   |      |  x   |        | &nbsp;                                                             |
```

Comment:
The `x` from the "not null" column must be removed. The `refEksternId` reference is not mandatory.

Reply:
Addressed in 7de1846